.gitignore

```diff
@@ -1,7 +1,7 @@
-__pycache__/
-*.pyc
-/.cache_metadata.json
-.coverage
-/.venv/
-/data/
-/cache/
+__pycache__/
+*.pyc
+/.cache_metadata.json
+.coverage
+/.venv/
+/data/
+/cache/

```

Comment:
What changed in this file? GitHub is not showing me anything :D

Reply:
shouldn't be different. Actually, scratch that. Might be line-endings (which is weird, vscode normally always commits as \n, not \r\n...)

Reply:
Yeah, no it should be LF 
![image](https://user-images.githubusercontent.com/14336407/150200169-6d737c3f-754b-41e5-955b-19ed6078ec97.png)

Reply:
I'll just pick that file from upstream to be sure

Reply:
I will check it on my end too :) Who ever invented `\r` ...... :P